openmetadata-service/src/main/resources/elasticsearch/en/test_suite_index_mapping.json

```diff
@@ -0,0 +1,86 @@
+{
```

Comment:
@07Himank we need to add this index mapping in zh and jp languages as well. Make sure you use appropriate stemmers and settings there

Reply:
yeah sure will do it

Reply:
handle in latest commit